posts/how-to-add-open-source-software-to-sourcegraph-cloud-video.md

```diff
@@ -1,7 +1,8 @@
 ---
-title: Video — How to add open source software to Sourcegraph cloud
+title: How to add open source software to Sourcegraph cloud (video)
```

Comment:
Still not sure how to approach the different media formats, but trying to keep it consistent across records for now